

In file included from arch/i386/mach-default/topology.c:30:
include/linux/nodemask.h: In function `__first_node':
include/linux/nodemask.h:219: warning: implicit declaration of function `min_t'


Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/linux/nodemask.h |    1 +
 1 files changed, 1 insertion(+)

diff -puN include/linux/nodemask.h~nodemask-build-fix include/linux/nodemask.h
--- 25/include/linux/nodemask.h~nodemask-build-fix	2004-08-02 20:45:06.652539888 -0700
+++ 25-akpm/include/linux/nodemask.h	2004-08-02 20:45:15.830144680 -0700
@@ -73,6 +73,7 @@
  *    inside a macro, the way we do the other calls.
  */
 
+#include <linux/kernel.h>
 #include <linux/threads.h>
 #include <linux/bitmap.h>
 #include <linux/numa.h>
_
